src/utils/deconflictChunk.ts

```diff
@@ -227,10 +224,22 @@ function deconflictTopLevelVariables(
 					(variable instanceof ExportDefaultVariable && variable.getOriginalVariable() !== variable)
 				)
 			) {
+				const cachedSafeVariableName = Object.getOwnPropertyDescriptor(
+					module.info.safeVariableNames,
+					variable.name
+				)?.value;
```

Comment:
I understand you are doing it like this because you do not want to get inherited properties. However, if we use an object without prototype or a Map, we could use simple property access (where I think the object without prototype is probably faster).